src/components/AllServicesDropdown/AllServicesDropdown.scss

```diff
@@ -102,7 +106,12 @@
           --pf-v6-c-tabs__link--PaddingLeft: var(--pf-t--global--spacer--md);
           --pf-v6-c-tabs__item--m-current__link--Color: var(--pf-t--global--text--color--link--default);
           --pf-v6-c-tabs--m-vertical--MaxWidth: 100%;
-          --pf-v6-c-tabs--m-vertical__list--before--BorderColor: transparent;
+        }
+        .pf-v6-c-tabs_item {
+          padding-inline-start: 100px !important;
```

Comment:
Are we able to write more specific rule without `!important` clause?

Reply:
not used. removed